### PR TITLE
indexer v1.1.41

### PR DESCRIPTION
**Pull request by miiu96**
Indexer v1.1.41

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm/v1_4 v1.4.34-rc7
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/covalent-indexer-go v1.0.6
-	github.com/ElrondNetwork/elastic-indexer-go v1.1.40
+	github.com/ElrondNetwork/elastic-indexer-go v1.1.41-0.20220419195249-74671c9b0c6c
```
Comment (bogdan-rosianu) [inline go.mod:11]: use proper release

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

**Review by dragos-rebegea — DISMISSED** [review]

File: go.mod (hunk 1 above)
Comment (miiu96) [reply go.mod]: done

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by iulianpascalau — COMMENTED** [review]
Maybe retarget this to feat april as this is not critical for all nodes?

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by popenta — APPROVED** [review]
@@                            Log scanner                                  @@
##                            master-with-indexer-v1.1.41                  ##
================================================================================
- Known Warnings              255
- New Warnings                172
- Known Errors                0
- New Errors                  0
- Panics                      0
================================================================================

**Pull request merged**